sample-linux.ini

```diff
@@ -10,6 +10,10 @@ maxCacheEntries = 2000000
 mruSymbolStateFile = /tmp/snappy-mru-symbols.json
 maxMRUSymbolsPersist = 10
 
+[DiskCache]
+cachePath = /tmp/snappy/cache
+maxCacheFiles = 50
```

Comment:
do you mean the raw .sym files are 15MB? How big are XULs after only the relevant lines are pickled?